extensions/functions_datetime.yaml

```diff
@@ -193,3 +203,66 @@ scalar_functions:
           - value: interval_year
           - value: interval_year
         return: boolean
+  - name: "assume_timezone"
+    description: Convert local time to UTC using given local time's timezone.
+    impls:
+      - args:
+          - options: [ TIMEZONE ]
+            name: Timezone of provided local time.
+            required: true
+          - value: timestamp
+        return: timestamp_tz
+  - name: "strptime"
+    description: Parse string into timestamp/date/time using provided format.
+    impls:
+      - args:
+          - options: [ FORMAT ]
```

Comment:
Same as above, this seems to be a misunderstanding of options.

Reply:
Most likely. If options are enums, how do I bring in a string parameter?